tap_github/client.py

```diff
@@ -110,6 +112,14 @@ def _request_with_backoff(
             )
             return response
 
+        # Rate limiting
+        if response.status_code == 403 and "Rate Limit Exceeded" in str(
+            response.content
+        ):
+            # Change token and force request retry.
+            self.github_authenticator.get_next_auth_token()
+            raise RuntimeError("Rate Limit Exceeded. Updating active token.")
```

Comment:
I don't understand these 2 lines. If you successfully rotate the token, you end up raising here, doesn't that interrupt the tap?

Reply:
This is in a function that has a backoff mechanism, so it changes the token, then fails to force the function to rre-run. I'll add a note